tests/test_unit_checking.py

```
@@ -0,0 +1,92 @@
+    ('Mt.steel ', SINGLE_ELEMENT, False),  # period not allowed
+    ('PJ / day', SINGLE_ELEMENT, False),  # no slash char
+    ('PJ    / (kT)', RATIO_ELEMENT, True),
+    ('(PJ) / (kT)', RATIO_ELEMENT, True),  # numerator optionally parenthesized
+    ('PJ / kT', RATIO_ELEMENT, False),  # no parens on denom
+    ('kWh/day/(cycle)', RATIO_ELEMENT, False),  # ambiguous slash char
+    ('(kWh/day)/(cycle)', RATIO_ELEMENT, True),
+]
+
+
+@pytest.mark.parametrize(
+    'entry, units_format, expected',
```

Ruff (0.14.7)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)